.github/workflows/scorecard.yaml

```diff
@@ -0,0 +1,45 @@
+name: "Scorecard tests"
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  scorecard-tests:
+    name: Scorecard test
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        kube-version:
+          - "1.21"
+          - "1.25"
+
+    steps:
+
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v3
+
+      - name: "install kuttl and kind"
+        run: ./hack/install/install-kuttl.sh
+
+      - name: "start kind"
+        env:
+          KUBE_VERSION: ${{ matrix.kube-version }}
+        run: kind create cluster --config kind-$KUBE_VERSION.yaml
+
+      - name: "wait until cluster is ready"
+        run:  kubectl wait --timeout=5m --for=condition=available deployment/coredns -n kube-system
```

Comment:
Is this really needed? When the `kind` command exists, the cluster should be up and running

Reply:
Yes, sometimes the kind takes longer to be available.